controllers/seedimage_controller.go

```diff
@@ -448,12 +448,6 @@ func (r *SeedImageReconciler) getRancherServerAddress(ctx context.Context) (stri
 
 func fillBuildImagePod(name, namespace, buildImg, baseImg, configMap string, pullPolicy corev1.PullPolicy, deadline int32) *corev1.Pod {
 	const volLim = 4 * 1024 * 1024 * 1024 // 4 GiB
-	const volRes = 2 * 1024 * 1024 * 1024 // 2 GiB
```

Comment:
@fgiudici I am not sure we actually need that, I'd say copying everything directly to the shared volume is just enough.

I faced this because if pulling the ISO from a container I need to share it to the builder container and the obvious way to do so is simply reusing the one we already have to store the built ISO.

Reply:
well, this refers to the same volume, it is just the minimum disk space we require for the Pod to be created.
This is mainly for the kube API and the kubelet to ensure the destination node will have this minimum required disk space available.
If you drop it, it would default to 'volLim', which is the *maximum* space that can be used on the storage.
That means anyway, that for each seedimage pod running the kubelet will compute 'volLim' disk space as reserved and no more available on the node for other workloads.
I really think we should keep this, otherwise may affect pod allocation.